src/hssm/config.py

```diff
@@ -12,8 +12,8 @@
 from ._types import LogLik, LoglikKind, SupportedModels
 from .defaults import (
     default_model_config,
-    get_default_model_meta,
 )
+from .modelmeta import get_default_model_meta
```

Comment:
late maybe but small question: Why are we calling this `meta`? Is this class lingo?